packages/core/src/command.tsx

```diff
@@ -83,14 +83,15 @@ export function apply(ctx: Context, config: Config) {
         }
         url ||= image.url
 
-        if (config.asset && ctx.assets) {
-          url = await ctx.booru.imgUrlToAssetUrl(url)
+        if (config.base64 || config.autoResize) {
```

Comment:
What is your plan if the target platform does not support sending base64 images? (e.g., adapter-qq)